RPIPs/RPIP-46.md

```diff
@@ -1,108 +0,0 @@
-10. Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
-    1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), or using higher `voter_share`
-    2. If higher `voter_share` is selected, the entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
-    3. The selected mechanism MAY be implemented in Saturn 1
-       1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.
-    4. The selected mechanism MUST be implemented in Saturn 2
-    5. When implemented, `surplus_share` SHOULD be renamed to something more specific
-
-## Specification taking effect with Saturn 2
-### Updating `voter_share`:
-The `voter_share` SHALL be adjustable, to target a share of voting eligible RPL (including 
-megapool staked RPL and legacy staked RPL) relative to the total supply of RPL minus RPL 
-owned by the protocol in its treasury or as a result of `surplus_share` buy backs. The voting
-eligible RPL target is a range between `vote_eligible_target_min` and `vote_eligible_target_max`
-1. Updating `voter_share`:
-   1. A new function SHALL be available to update `voter_share`, which MAY be called by anyone
-   2. It MUST revert if it's been called within the last 45 days
-   3. It MUST revert if the total RPL eligible to vote is between `vote_eligible_target_min` and `vote_eligible_target_max` (inclusive)
-   4. If <`vote_eligible_target_min` of total RPL is eligible to vote and the function succeeds:
-      1. `voter_share` is increased to `voter_share * (1+voter_share_relative_step)`
-      2. `surplus_share` is decreased by the difference between the old and new `voter_share`
-         1. If this would reduce `surplus_share` below 0%, the function call MUST revert 
-   5. If >`vote_eligible_target_max` of total RPL is eligible to vote and the function succeeds:
-      1. `voter_share` is decreased to `voter_share / (1+voter_share_relative_step)`
-      2. `surplus_share` is increased by the difference between the old and new `voter_share`
-   6. Because this involves _voters_ modifying `voter_share`, there is an acknowledged conflict of interest here. As a result, changing this method of "Updating `voter_share`" SHALL require a supermajority vote with at least 75% of the vote in support of any change.
-3. `voter_share_relative_step`, `vote_eligible_target_min`, and `vote_eligible_target_max` MAY be updated by pDAO vote; however, it SHALL require a supermajority vote with at least 75% of the vote in support of any change.
-4. The initial settings SHALL be:
-   1. `voter_share_relative_step`: 15%
-   2. `vote_eligible_target_min`: 55%
-   3. `vote_eligible_target_max`: 65%
-
-## Optional heuristics
```

Comment:
I agree with Samus this should be removed. I think ideal is to move it to a supporting document that's linked and present in the repo.

Reply:
If we move it to somewhere less visible, it is gone for most intents and purposes. Eg, we have explicit documentation about our assumptions when choosing to go with 2.4ETH required for LEB8s. Our assumptions were violated and we took no action (and frankly didn't even explicitly realize it for many months).

In my eyes, it is very important to the idea of UARS that we _actively_ listen and react to the market, and exemplifying that is the point of this section.

Reply:
If this section is that important. Should it become actual specification with SHOULD statements instead? eg:
"If the NO queue is continuously over 500 deposits for 2 weeks and the trend is upwards, the pDAO SHOULD act to either increase rETH demand or decrease NO supply via one or more of the following actions..."

Reply:
They are meant to be truly optional (ie, we may learn better when we wish to act and how). OPTIONAL is an RFC2119 imperative similar to MAY, which is what I was trying to aim for her... I would love some help better communicating.

Reply:
If we do keep this section in the actual RPIP then I think the disclaimers already present are good enough:
> "This section reflects some of the thinking at the time this RPIP was drafted. These ideas are explicitly not binding/enforceable, and they may freely change over time/context."... 

This can be a section we revisit before Saturn goes live and discuss further among nerds. If someone thoughtfully disagrees we should consider if there are better abstract guidelines to replace here, but since they are non-binding/enforceable it shouldn't block any work/time to market